pkg/router/template/template_helper_test.go

```diff
@@ -1061,9 +1098,24 @@ func TestParseIPList(t *testing.T) {
 				}
 				return
 			}
-			if got != tc.input {
+			if tc.expectedEmpty && got != "" || got != tc.expectedReturn {
 				t.Errorf("Failure: expected %q, got %q", tc.input, got)
 			}
 		})
 	}
+
+	for _, tc := range testCases {
```

Comment:
This version of `for` looks good to me. Can you remove the previous one (starts on the line 1055)? Because correctly we run the test for `parseIPList` twice?